Fix build failure with Xcode 9 and runtime crashes on High Sierra.
https://github.com/textmate/textmate/commit/234e64dcfcf2845d3a1c34e29bcd7d5419d7f42d
--- target.orig
+++ target
@@ -13,9 +13,9 @@ FLAGS += -fcolor-diagnostics
 FLAGS += -DNDEBUG -Os
 # LINK   = OakDebug
 
-CXX_FLAGS    += -fvisibility=hidden -std=c++1z -stdlib=libc++
+CXX_FLAGS    += -fvisibility=hidden -std=c++14 -stdlib=libc++
 OBJC_FLAGS   += -fvisibility=hidden -fobjc-arc -std=c99 -fobjc-abi-version=3
-OBJCXX_FLAGS += -fvisibility=hidden -std=c++1z -stdlib=libc++ -fobjc-abi-version=3
+OBJCXX_FLAGS += -fvisibility=hidden -std=c++14 -stdlib=libc++ -fobjc-abi-version=3
 OBJCXX_FLAGS += -fobjc-arc -fobjc-call-cxx-cdtors
 LIBS         += c++
 
